bgstally/formatters/clb.py

```diff
@@ -3,168 +3,122 @@
 import traceback
 import re
 from bgstally.activity import STATES_ELECTION, STATES_WAR, Activity
-from bgstally.constants import CheckStates, DiscordActivity
+from bgstally.constants import CheckStates, DiscordActivity, DiscordPostStyle
 from bgstally.debug import Debug
 from bgstally.formatters.default import DefaultActivityFormatter
 #from bgstally.formatters.base import FieldActivityFormatterInterface
-from bgstally.utils import _, __, human_format, is_number
+from bgstally.utils import _, __, human_format, is_number, catch_exceptions
 from thirdparty.colors import *
 
 class CLBActivityFormatter(DefaultActivityFormatter):
-    """Activity formatter that outputs Lorum Ipsum
+    """
+    Activity formatter that outputs in a format suitable for Celestial Light Brigade Discord channel
     """
 
     def __init__(self, bgstally):
-        """Instantiate class
```

Comment:
You seem to have ripped out a lot of function comments, including parameter defs, was this intentional?